### PR TITLE
chore(just): clean up ujust recipes to fit newer style

### LINKED ISSUE #1805: `ujust switch-channel` alias is broken
This is probably something simple I'm missing.

https://github.com/ublue-os/bluefin/blob/71a923264e334612d3d90dff3e2b196f560eacbc/just/bluefin-system.just#L262

### PR DESCRIPTION
**Pull request by tulilirockz**
Should fix #1805

### CODE REVIEW

**Comment by m2Giles** [comment]
We should also use this as an opportunity to clean up stale ones.

File: just/bluefin-system.just [hunk 1]
```diff
@@ -253,8 +254,14 @@ install-system-flatpaks:
 configure-grub:
     @/usr/libexec/configure-grub.sh
 
-alias rollback-helper := rebase-helper
-alias switch-channel := rebase-helper
+switch-channel:
```
Comment (castrojo) [inline just/bluefin-system.just:257]:
```suggestion
switch-stream:
```

File: just/bluefin-system.just [hunk 2]
```diff
@@ -253,8 +254,14 @@ install-system-flatpaks:
...
+  @ujust rebase-helper
+
+switch-channels:
```
Comment (castrojo) [inline just/bluefin-system.just:260]:
```suggestion
switch-streams:
```

Comment (castrojo) [reply just/bluefin-system.just:260]: coreos calls these streams

**Review by castrojo — CHANGES_REQUESTED** [review]

**Comment by m2Giles** [comment]
when working with the justfiles. Please be sure to run `just fix`. We run `just check` as part of the workflow to make sure we don't ship broken justfiles.

**Comment by tulilirockz** [comment]
Should I also include a move to bootc on this PR? I am thinking about refactoring most of these scripts to not use rpm-ostree and instead use a wrapper over bootc that allows it to run rootless (ubootc or something)

**Comment by tulilirockz** [comment]
The indentation on some places also seems a bit funky, ill be reformatting them to always fit 4 spaces indentation

**Comment by castrojo** [comment]
> Should I also include a move to bootc on this PR? 

Follow on PRs are always better instead of conflating things. 

> a wrapper over bootc that allows it to run rootless (ubootc or something)

This is out of scope for us, we're trying to delete as much of our custom stuff as possible these days.

File: just/bluefin-system.just [hunk 3]
```diff
@@ -253,20 +272,35 @@ install-system-flatpaks:
...
-alias rollback-helper := rebase-helper
-alias switch-channel := rebase-helper
+# Rebase assistant
+[group('System')]
+switch-stream:
+    @ujust rebase-helper
+
+# Rebase assistant
+[group('System')]
+switch-streams:
```
Comment (m2Giles) [inline just/bluefin-system.just:282]: just alias this.

**Comment by tulilirockz** [comment]
Then we could regroup stuff, honestly I feel like separating it in `apps`, `tools` and `system` doesnt seem like the best? `apps` and `tools` could be the same category (`tools`)

**Comment by castrojo** [comment]
We need to keep jetbrains (people comment about how useful this is all the time), and the tailscale toggle was an ask in the survey. So let's keep that for people who use other VPNs. Not sure on invokeai so let's keep it for now. We can always do another pass.

**Comment by castrojo** [comment]
OK one final pass from others would be appreciated on this one! (I know m2 wanted one more look).

**Comment by tulilirockz** [comment]
Alright, so, after manually testing every single one of the affected recipes, they all seem to work fine, without any major changes. The only ones that I saw that were broken were the toggle-tailscale and setup-ollama ones - just reverted them. The tailscale one also didnt seem to toggle the extension on the `Silverblue` $VARIANT so I just made it uppercase :p

**Review by castrojo — APPROVED** [review]

**Pull request merged**